microk8s-resources/wrappers/apiservice-kicker

```diff
@@ -92,6 +96,18 @@ do
       fi
     fi
 
+    if snapctl services microk8s.daemon-cluster-agent | grep active >/dev/null
```

Comment:
Now that we do the restart on the config hook do we still need the apiserver-kicker to change?